### DISCUSSION TITLE
React 17

### DISCUSSION THREAD

**Discussion by rpyyj** [dowjones/react-dropdown-tree-select#518] [Q&A]
Hi! Thank you for the library. Do you have a timeline for when it will be ready for React 17? I'm upgrading a project and would like to keep `react-dropdown-tree-select` if possible, but the UNSAFE_ issue prevents that.

Yes, I've seen this: https://github.com/dowjones/react-dropdown-tree-select/issues/292

I noticed the stale `hooks` branch and an outstanding PR to convert to functional components. Any thoughts on when you might get back to it? If it's not in the cards, do you or any other users have recommendations on a replacement?

**Comment by mrchief** [comment]
Hey, thanks for the interest. It's definitely on the cards but what it lacks is time on my part.
